src/utils/file_utils.js

```diff
@@ -74,3 +75,15 @@ export function parseClientIdsFromFormData(formdata) {
 
     return clientIds;
 }
+
+export function getFileUrl(fileId) {
```

Comment:
nothing to do here, @jarredwitt maybe we could replace the ones in RN for this methods?